src/main/java/com/pscheduler/server/repository/GPACourseRepository.java

```diff
@@ -0,0 +1,24 @@
+package com.pscheduler.server.repository;
+
+import com.pscheduler.server.model.GpaCourse;
```

Comment:
Some unneeded imports. Lets remove the search/query for now and try to just have the model and API working to be merged.

Reply:
Done